role.tf

```diff
@@ -25,7 +25,7 @@ data "aws_iam_policy_document" "this" {
 // This policy enables SSM on the instance
 resource "aws_iam_role_policy_attachment" "ssm" {
   role       = aws_iam_role.this.name
-  policy_arn = "arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore"
+  policy_arn = "arn:aws:iam::aws:policy/service-role/AmazonEC2RoleforSSM"

```

Comment:
Looks like this policy is being deprecated: https://docs.aws.amazon.com/aws-managed-policy/latest/reference/AmazonEC2RoleforSSM.html

Reply:
This was my mistake; it looks like the existing policy has everything it needs.
Let's revert this line.

Reply:
Ah gotcha, done!